Sources/KeyboardManager/KeyboardManager.swift

```diff
@@ -287,7 +296,12 @@ open class KeyboardManager: NSObject, UIGestureRecognizerDelegate {
         frame.size.height = window.bounds.height - frame.origin.y
         keyboardNotification.endFrame = frame
 
-        let yCoordinateDirectlyAboveKeyboard = -frame.height
+        var yCoordinateDirectlyAboveKeyboard = -frame.height
+
+        if shouldApplyAdditionBottomSpaceToInteractiveDismissal {
```

Comment:
Would it be better to add check for the `additionalBottomSpace?()` here too so we don't have `?? 0` fallback?